lib/dialogs/logout-confirmation/index.tsx

```diff
@@ -32,56 +34,45 @@ export class LogoutConfirmation extends Component<Props> {
 
     return (
       <div className="logoutConfirmation">
-        <Dialog onDone={closeDialog} title="Unsynchronized Notes Found">
-          <section>
-            <h1>Are you sure you want to logout?</h1>
-          </section>
-
+        <Dialog onDone={closeDialog} title="Unsynced Notes">
+          <p className="explanation">
+            {notes.size > 0
+              ? 'Logging out will delete any unsynced notes.'
```

Comment:
Not sure if anyone else has feelings about this but I find the use of the word form `unsynced` a bit too casual for official use. Any thoughts about "unsynchronized" as a replacement? Am I over-thinking it?

Reply:
I agree.